index.js

```diff
@@ -212,6 +212,11 @@ RainbowConfig.declareOption(
   'The challenge window during which anyone can challenge an incorrect ED25519 signature of the Near block, in EthClient, in seconds.',
   14400
 )
+RainbowConfig.declareOption(
```

Comment:
Please also add it to `init-near-contracts` list.

Reply:
You probably meant `init-eth-client`. Added.